tests/t525-operator.c

```diff
@@ -105,6 +105,13 @@ int main(int argc, char **argv) {
     printf("Incorrect context data for other: %f != 2.0", ctx_data_2.other);
   // LCOV_EXCL_STOP
 
+  // Check requesting label for field that doesn't exist returns NULL
```

Comment:
One of these days we should set up a CI job to run our test suite with a simple CPU backend against Valgrind to check for leaks.

Reply:
Would `-faddress=sanitize` have caught this? It runs much faster than Valgrind and somewhat avoids needing to build a special MPICH to be valgrind-clean.

Reply:
I'm not sure; I can go back in history and check. I was thinking of running Valgrind only on the core library tests to bypass all the MPICH, LIBXSMM, CUDA, etc noise.